test/features/user_records_an_application_test.rb

```diff
@@ -29,10 +31,12 @@ def test_an_application_with_no_company_is_rejected
   end
 
   def test_viewing_the_details_of_an_application
-    app = Application.create!(:company => "Basecamp",
+    user = create(:person)
+    app = user.applications.create!(:company => "Basecamp",
                               :url => "http://basecamp.com",
-                              :status => "open")
+                              :status => "applied")
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.